### PR TITLE
docs: Improve some zh-tw translations in download/package-manager.md

### PR DESCRIPTION
**Pull request by PeterDaveHello**
cc @nodejs/nodejs-tw

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@bluet @jackycute not sure if you guys would like to help take a look/review 😃

File: locale/zh-tw/download/package-manager.md [hunk 1]
```diff
@@ -141,32 +141,32 @@ choco install nvs
...
 
-添加最新 LTS 版本的 node：
+新增最新 LTS 版本的 node：
 
 ```bash
 nvs add lts
 ```
 
-然後運行 `nvs use` ，為當前腳本的 `PATH` 路徑下添加一個 node 版本：
+然後執行 `nvs use` ，為當前腳本的 `PATH` 路徑下新增一個 node 版本：
```
Comment (jackycute) [inline locale/zh-tw/download/package-manager.md]:
```suggestion
然後執行 `nvs use` ，為目前 shell 的 `PATH` 路徑下新增一個 node 版本：
```

File: locale/zh-tw/download/package-manager.md [hunk 2]
```diff
@@ -141,32 +141,32 @@ choco install nvs
 ```
 
 #### macOS，UnixLike
-你可以在 [此處](https://github.com/jasongin/nvs/blob/master/doc/SETUP.md#mac-linux) 找到關於在 macOS / 形如 Unix 操作系統的安裝步驟文檔。
+你可以在 [此處](https://github.com/jasongin/nvs/blob/master/doc/SETUP.md#mac-linux) 找到關於在 macOS / 形如 Unix-like 作業系統的安裝步驟文件。
```
Comment (jackycute) [inline locale/zh-tw/download/package-manager.md]:
```suggestion
你可以在 [此處](https://github.com/jasongin/nvs/blob/master/doc/SETUP.md#mac-linux) 找到關於在 macOS / Unix-like 作業系統的安裝步驟文件。
```

**Review by jackycute — CHANGES_REQUESTED** [review]
Would love to give a review @PeterDaveHello.
I've made some suggestion according to the en version.
https://github.com/nodejs/nodejs.org/blob/master/locale/en/download/package-manager.md

**Review by jackycute — APPROVED** [review]
LGTM.

**Review by bluet — APPROVED** [review]
LGTM

**Comment by PeterDaveHello** [comment]
Thank you guys 😄

**Pull request merged**